### PR TITLE
First cut string extraction

### PR DESCRIPTION
**Pull request by samzx**
First pass to make sure integration with zendesk translation service is working properly.

This required to be merged before: https://github.com/zendesk/rosetta/pull/2975

## Description
We extracted some strings to test functionality. This is the first pass, will mainly go unnoticed to users. If the translation service works, we'll be able to pull the Japanese translations and import it into this repo.

## References
[Github Issue](https://github.com/zendesk/volunteer_portal/issues/167)

## CCs
@ovillegaszen 
@zendesk/volunteer 

## Risks (if any)
* low - affects strings

### CODE REVIEW

File: app/javascript/components/Calendar/__snapshots__/test.js.snap [hunk 1]
```diff
@@ -1,31 +1,7 @@
...
-    views={
-      Array [
-        "month",
-      ]
-    }
-  />
-</Layout>
+<Consumer>
+  <Component />
+</Consumer>
```
Comment (samzx) [inline app/javascript/components/Calendar/__snapshots__/test.js.snap]: This probably happened when wrapping the calendar component with `withNamespaces`

File: app/javascript/components/Calendar/index.js [hunk 2]
```diff
@@ -21,7 +23,7 @@ const styles = {
 }
 
 // Custom components given to BigCalendar
-const calendarComponents = (currentUser, offices, filters, filterActions) => {
+const calendarComponents = (currentUser, offices, filters, filterActions, t) => {
```
Comment (ThuTrinh) [inline app/javascript/components/Calendar/index.js]: would be good practice to give `t` a more descriptive name for readability.

File: app/javascript/components/Calendar/index.js [hunk 3]
```diff
@@ -145,11 +148,18 @@ const Calendar = ({
         events={selectEvents(events, currentUser, filters)}
         eventPropGetter={eventPropGetter}
         views={['month']}
-        components={calendarComponents(currentUser, offices, filters, {
-          changeShowFilter,
-          changeEventFilter,
-          changeOfficeFilter,
-        })}
+        culture="en"
```
Comment (ThuTrinh) [inline app/javascript/components/Calendar/index.js]: is this the culture default value or is it hard coded?

File: app/javascript/components/Toolbar/index.js [hunk 4]
```diff
@@ -94,33 +94,52 @@ const OfficeFilter = ({ value, onChange, offices }) => (
...
       <button className={s.btn} type="button" onClick={() => onNavigate(navigate.NEXT)}>
-        ›
+        {t('dashboard.layoutdatetab.after')}
       </button>
     </div>
     <div className={s.filterBar}>
       <div className={s.filterDropdowns}>
-        <ShowFilter value={showFilter.value} onChange={(_e, _i, value) => showFilter.onChange(value)} />
-        <EventFilter value={eventFilter.value} onChange={(_e, _i, value) => eventFilter.onChange(value)} />
+        <ShowFilter t={t} value={showFilter.value} onChange={(_e, _i, value) => showFilter.onChange(value)} />
```
Comment (ThuTrinh) [inline app/javascript/components/Toolbar/index.js]: `t` as a component property is not very descriptive. This makes it hard to quickly obtain context or understand how to set this value.

File: app/javascript/components/Calendar/index.js (hunk 3 above)
Comment (samzx) [reply app/javascript/components/Calendar/index.js]: we're just explicitly setting the culture here so in the future we can change it via an interface / menu. But I think "en" is the default

File: app/javascript/components/Calendar/index.js (hunk 2 above)
Comment (samzx) [reply app/javascript/components/Calendar/index.js]: I think it's standard to have `t` as the translation function ie. the API gives the example: https://www.i18next.com/overview/api

What do you think?

Comment (ThuTrinh) [reply app/javascript/components/Calendar/index.js]: yeah , I wasn't aware that this prop was added by a third-party. Ignore me :)

**Review by ThuTrinh — APPROVED** [review]

**Review by ThuTrinh — APPROVED** [review]

**Pull request merged**